main.go

```diff
@@ -18,7 +18,10 @@ func main() {
 
 	e.Use(middleware.Static("frontend/dist"))
 
-	pkg.RouterHandler(e.Group("/api"))
+	router.RegisterApi(e)
+	if os.Getenv("TWP_ENV") == "dev" {
```

Comment:
it is better to make `"dev"` into a const or something for better readability and prevent typos
example:
```go
type Environment string

const (
    DEV Environment = "dev"
    PROD Environment = "prod"
)
```

Reply:
+ Add new package environment for env variables